lib/index.js

```diff
@@ -562,6 +562,33 @@ class Instagram {
       `accounts/access_tool/current_follow_requests?__a=1${cursorParam}`
     )
   }
+  
+  async getUserMediaTagged({id, first, after}){
+    return this.request.get('/graphql/query/', {
+      qs: {
+        query_hash: 'ff260833edf142911047af6024eb634a',
+        variables: JSON.stringify({
+          id: id,
```

Comment:
Linter blames on these lines. Why don't you want to use 
`        variables: JSON.stringify({
          id,
          first,
          after
        })`